portfolio/src/main/webapp/script.js

```diff
@@ -28,7 +28,19 @@ function addRandomQuoteOrFact() {
 }
 
 function hello(){
-   fetch('/data').then(response=>response.text()).then((hi)=>{
-    document.getElementById('response').innerText = hi;
-  });
+   fetch('/data').then(response=>response.json()).then((comment)=>{
+    const commentList = document.getElementById('response');
+    comment.forEach((com) => {commentList.appendChild(createListElement(com));
+    });
+    console.log(com);
```

Comment:
For betting debugging I think you should do `console.log(comment)` right under line 31 instead of this. Since in this case `com` is going to be undefined because you're outside of the scope of it.